FLIR/conservator/cli/managers.py

```diff
@@ -163,11 +163,26 @@ def list_media(identifier, recursive):
             return True
 
         @group.command(
-            help="Download a Collection to the current directory, or the specified path."
+            help="""
+            Download a Collection to the current directory, or to the specified
+            path.
+
+            If this operation is interrupted, repeat the command to resume
+            where the previous run left off.  Partially-downloaded or
+            corrupted files will be downloaded again.  Datasets will only be
+            re-downloaded if the `--overwrite_datasets` option is supplied.
+            """
         )
         @click.argument("identifier")
         @click.argument("localpath", default=".")
         @click.option("-d", "--datasets", is_flag=True, help="Pull datasets")
+        @click.option(
+            "-o",
+            "--overwrite_datasets",
```

Comment:
Please use a hyphen instead of an underscore when naming command line options.

Reply:
Done.